index.html

```diff
@@ -1113,6 +1113,59 @@ <h2>Fragment</h2>
 
       </section>
 
+      <section>
+        <h2>Relative DID URLs</h2>
+
+        <p>
+A relative DID URL is any URL value in a <a>DID document</a> that does not start
+with <code>did:&lt;method-name>:&lt;method-specific-id></code>. More
+specifically, it is any URL value that does not start with the ABNF defined  in
+Section <a href="#generic-did-syntax"></a>. The contents of the URL typically
+refers to a resource in the same <a>DID document</a>. Relative DID URLs MAY
+contain relative path components, query parameters, and fragment identifiers.
+        </p>
+
+        <p>
+When resolving a relative DID URL reference, the algorithm specified in <a
+data-cite="RFC3986#section-5">RFC3986 Section 5: Reference Resolution</a>  MUST
+be used. The <strong>base URI</strong> value is the <a>DID subject</a>.  The
```

Comment:
I think this is another example where we are conflating the DID value and the DID subject. We drove that out of the spec once. The DID subject is the thing identified by the DID, not the identifier value. So I think this should say that the base URI is the DID value. Is that accurate?

Reply:
Yes, will fix.